cppman/main.py

```diff
@@ -131,15 +141,20 @@ def rebuild_index(self):
                 for title in results:
                     for (k, a) in results[title]["aliases"]:
                         sql_results = self.db_cursor.execute('SELECT id, keyword FROM "%s_keywords" '
-                                                             'WHERE keyword LIKE "%s::%%" OR keyword LIKE "%s" OR keyword LIKE "%s %%" OR keyword LIKE "%s)%%" OR keyword LIKE "%s,%%"' % (table, k, k, k, k, k)).fetchall()
+                                                             'WHERE keyword LIKE "%%::%s::%%" OR keyword LIKE "%s::%%" OR keyword LIKE "%s" OR keyword LIKE "%s %%" OR keyword LIKE "%s)%%" OR keyword LIKE "%s,%%"' % (table, k, k, k, k, k, k)).fetchall()
```

Comment:
can we have a better line breaks here to keep the max col number to 80?

Reply:
Yes, of course.